packages/react-storage/package.json

```diff
@@ -71,7 +71,7 @@
       "name": "createStorageBrowser",
       "path": "dist/esm/browser.mjs",
       "import": "{ createStorageBrowser }",
-      "limit": "15.5 kB",
+      "limit": "17.4 kB",
```

Comment:
poked around the output and seems accurate, since the two main files I'm adding are 4kB before minification